core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -493,7 +493,7 @@ TEE_Result crypto_hash_final(void *ctx, uint32_t algo, uint8_t *digest,
 
 #if defined(_CFG_CRYPTO_WITH_ACIPHER)
 
-#define LTC_MAX_BITS_PER_VARIABLE   (4096)
+#define LTC_MAX_BITS_PER_VARIABLE	CFG_CORE_BIGNUM_MAX_BITS
```

Comment:
Since `LTC_MAX_BITS_PER_VARIABLE` is used only in this file, it's better to `s/LTC_MAX_BITS_PER_VARIABLE/CFG_CORE_BIGNUM_MAX_BITS/` in the whole file.